src/ZoneServer/Skills/Combat/SkillModifier.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Melia.Shared.Game.Const;
+
+namespace Melia.Zone.Skills.Combat
+{
+	/// <summary>
+	/// A class for properties that can modify the damage calculation for a skill
+	/// </summary>
+	public class SkillModifier
```

Comment:
The skill modifier class is a good idea, though I gotta admit that I dislike the amount of passing downs we have to do. I wonder if we could perhaps combine this with the skill or the skill hit result in some way, which we're already passing around anyway. Then again, I guess this _is_ most likely a one time change. Hm 🤔

Reply:
What do you mean by passing down?  You mean the number of methods that have to be adjusted to allow for it?

Something I was able to do here is that the skillmodifier is an overload, so if a method doesn't use it, they don't have to include it (a blank skillmodifier will be created that doesn't modify the skill's behaviour in any way).